### ISSUE TITLE
Cannot show Record with an undefined property

### ISSUE THREAD

**Issue by jonaskello** [andrewdavey/immutable-devtools#8]
When trying to inspect a record I sometimes get the error:

```
Custom Formatter Failed: Illegal format: obligatory attribute "object" isn't specified
```

I think it has to do with the fact that a property of the record is set to undefined but not sure...

**Comment by andrewdavey** [comment]
Should be fixed in 0.0.5

**Comment by jonaskello** [comment]
Thanks! I can't reproduce the conditions for my original issue but I did a test with a record with undefined values and that worked so I think it is fixed!
